extensions/sequelize/src/sequelize/sequelize.repository.base.ts

```diff
@@ -68,6 +68,27 @@ import {isTruelyObject} from './utils';
 const debug = debugFactory('loopback:sequelize:repository');
 const debugModelBuilder = debugFactory('loopback:sequelize:modelbuilder');
 
+interface InclusionWithRequired extends Inclusion {
+  /**
+   * Setting this option to true will result in an inner join query that
+   * explicitly requires the specified condition for the child model.
+   *
+   * @see https://loopback.io/pages/en/lb4/readmes/loopback-next/extensions/sequelize/#inner-join
+   */
+  required?: boolean;
+}
+
+type InclusionFilterWithRequired = string | InclusionWithRequired;
+
+interface FilterWithRequired<T extends object> extends Filter<T> {
+  include?: InclusionFilterWithRequired[];
+}
+
+type FilterWithRequiredExcludingWhere<T extends object> = Omit<
+  FilterExcludingWhere<T>,
```

Comment:
This won't have required flag, does it? 

---

```suggestion
  FilterWithRequired<T>,
```

Reply:
@shubhamp-sf Ah yes, thanks for catching this!

Reply:
Fixed: https://github.com/loopbackio/loopback-next/pull/10192/commits/e98997d24b8b134145f3b43e3dc4e21a25b72285